instrumentation/kamon-instrumentation-common/src/main/resources/reference.conf

```diff
@@ -45,6 +45,16 @@ kamon {
           enabled = yes
         }
 
+        # 
+        # Configuration for HTTP response handling
+        #
+        response {
+          # Controls adding custom HTTP headers to the responses
+          #   - none: Disabled, no additional headers are appended to the response
+          #   - fqcn: FQCN for a HttpServerResponseHeaderGenerator implementation
+          #
+          header-generator = "none"
```

Comment:
This setting should be moved to the `response-headers` section. That section already has the settings for including the trace and span ids, so this would make a lot more sense there.

Also, just as a nitpicking, I would user `headers-generator` (plural) as the setting name.

Reply:
I deliberately chose not to put it under `response-headers` due to it being under `tracing` and I didn't really think this feature was a tracing feature.
My concern is mainly with the functionality, where the config is and the name of it is all the same for me...:)
I made a suggestion, it's your source code so I'm fine to move it to different place.

Reply:
Hmm, see if I managed to do this `--amend` thingie correctly. Goal was to have a single commit to merge.
I'm no git wizard, it's all voodoo-hoodoo for me...:)

Reply:
Hey @pnerg, I think I overlooked the details here. Most likely you were right on placing the settings but now I can't go back and see how it was before! Can you please tell me where did you mean to have them in the first place? Thanks!

Reply:
that's the downside of squashing commits using `--amend`...history is gone.

Anyways I had an own section `server` in parallel with the `tracing` context like this.
instrumentation-common:
```
kamon.instrumentation.http-server.default { 
  server {
    response.headers-generator = "none"
  }
}
```
and in akka-http:
```
kamon.instrumentation.akka.http {
  server {
    #response.headers-generator = "none"
  }
}
```

Just decide where u guys want the config and I'll stuff it there...:)

Reply:
It smells a bit bad (at least to me) that the headers generator is under tracing, even though they are not limited to include trace data in the responses. But having all the settings related to tweaking response headers in a single place seems the most ergonomic solution to users so we are going to keep it under the `tracing.response-headers` section.

If we end up introducing a couple more response-related settings we might pull them into a dedicated section, but for now I think we are fine with leaving it as it is. Thanks a lot @pnerg!